sniffles/rulereader.py

```diff
@@ -1458,6 +1473,9 @@ def readRuleFile(self, filename):
         # rule files to be in a different format.
         parser = self.findParser(filename)
         parser.parseRuleFile(filename)
+        if isinstance(parser, PetabiRuleParser):
+            if parser.getBackgroundPercent():
```

Comment:
Do we really need it here.  Shouldn't we only get this if we are parsing a petabi-rule file?